core/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -120,6 +120,7 @@ grant {
   permission java.io.FilePermission "/proc/sys/vm/max_map_count", "read";
 
   // io stats on Linux
+  permission java.io.FilePermission "/proc/self/mountinfo", "read";
```

Comment:
This one is tricky. We use to not need this permission because we read this virtual file before the security manager is installed so we did not need permissions. Now, we only need to read this file after the security manager is installed so now we need to grant this permission otherwise we do not get disk I/O stats.